posthog/utils.py

```diff
@@ -396,33 +396,10 @@ def base64_decode(data):
     return data.decode("utf8", "surrogatepass").encode("utf-16", "surrogatepass")
 
 
-# Used by non-DRF endpoins from capture.py and decide.py (/decide, /batch, /capture, etc)
-def load_data_from_request(request):
-    data = None
-    if request.method == "POST":
-        if request.content_type in ["", "text/plain", "application/json"]:
-            data = request.body
-        else:
-            data = request.POST.get("data")
-    else:
-        data = request.GET.get("data")
-
+def load_data_from(data: Any, compression: str):
```

Comment:
naming nit: It sounds like all this is doing is decompressing data, so I'd propose naming this

```suggestion
def decompress(data: Any, compression: str):
```